js/components/tooltip.js

```diff
@@ -0,0 +1,115 @@
+var React = require('react');
+var deepmerge = require('deepmerge');
+
+var verticalOffset = 80;
+function getContainerStyle(bottom, visible, responsivenessMultiplier, alignment) {
+    var verticalAlignment = verticalOffset * responsivenessMultiplier;
+    var alignmentStyle = {
+        left: {
+            left: 0,
+            transform: 'translate(0,' + verticalAlignment + '%)'
+        },
+        center: {
+            left: '50%',
+            transform: 'translate(-50%,' + verticalAlignment + '%)'
+        },
+        right: {
+            right: 0,
+            transform: 'translate(0,' + verticalAlignment + '%)'
+        }
+    };
+
+    var style = {
+        position: 'absolute',
+        color: 'white',
+        fontFamily: '"Roboto Condensed", sans-serif',
+        opacity: visible ? '0.75' : '0',
+        fontWeight: 'normal',
+        bottom: bottom,
+        transition: '1s',
+        visibility: visible ? 'visible' : 'hidden',
+    }
+
+    return deepmerge(alignmentStyle[alignment], style);
+
+}
+
+function getBoxStyle(responsivenessMultiplier) {
+    return {
+        borderRadius: '3px',
+        fontSize: '15px',
+        textShadow: 'none',
+        background: 'black',
+        paddingTop: 8 * responsivenessMultiplier,
+        paddingRight: 20 * responsivenessMultiplier,
+        paddingBottom: 8 * responsivenessMultiplier,
+        paddingLeft: 20 * responsivenessMultiplier
+    }
+}
+
+var pointerAlignment = {
+    left: '10%',
+    center: '45%',
+    right: '85%'
+}
+function getPointerStyle(alignment) {
+    return {
+        position: 'absolute',
+        borderLeft: '5px solid transparent',
+        borderRight: '5px solid transparent',
+        borderTop: '5px solid black',
+        left: pointerAlignment[alignment]
+    };
+}
+
+
+var Tooltip = React.createClass({
+    render: function () {
+        if (this.props.enabled) {
+            return (
+                <div style={{ position: 'relative' }}>
+                    <div className='oo-tooltip' style={getContainerStyle(this.props.bottom, this.state.visible, this.props.responsivenessMultiplier, this.props.alignment)}>
+                        <div style={getBoxStyle(this.props.responsivenessMultiplier)}>
```

Comment:
We should add class names to these inner elements. Customers should still be able to easily customize with CSS if they choose to.